scripts/test_workflow.sh

```diff
@@ -69,7 +69,7 @@ check_ret $? "Test Couler failed"
 ############# Run SQLFLow test with Argo Mode #############
 function test_workflow() {
     # start a SQLFlow MySQL Pod with testdata
-    kubectl run mysql --port 3306 --env="SQLFLOW_MYSQL_HOST=0.0.0.0" --env="SQLFLOW_MYSQL_PORT=3306" --image=sqlflow/sqlflow --command -- bash /start.sh mysql
+    kubectl run mysql --port 3306 --env="SQLFLOW_MYSQL_HOST=0.0.0.0" --env="SQLFLOW_MYSQL_PORT=3306" --image=${SQLFLOW_WORKFLOW_STEP_IMAGE} --command -- bash /start.sh mysql
```

Comment:
Maybe we can set the default docker image by: `${SQLFLOW_WORKFLOW_STEP_IMAGE:-sqlflow/sqlflow}` .